assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -184,12 +187,14 @@ Title.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node,
   ]),
+  canCollapse: PropTypes.bool,
 };
 
 Title.defaultProps = {
   isPrimary: false,
   isSecondary: false,
   isResizable: false,
+  displayCollapse: true,
```

Comment:
This should be removed, no? As you already added the default inline. Also, it's the wrong name here anyway.